### PR TITLE
✨ Source Google Analytics (Universal Analytics): disable connectors

### PR DESCRIPTION
**Pull request by maxi297**
## What
The API will be deprecated on July 1st. We need to:
* Deprecate the connector
* Prevent the connector from sending alerts

## How
* Setting the registries.x.enabled to false
* Have the main.py just break with a config error
    * Note that I did not remove the code which could be confusing for future dev. We can do that if we see value in it. Else, I would wait for us to `rm -rf source-google-analytics-v4`

The outcome looks likes this:
```
(.venv) source-google-analytics-v4% python main.py spec
{"type": "SPEC", "spec": {"connectionSpecification": {}}}

(.venv) source-google-analytics-v4% python main.py check --config secrets/config.json
{"type": "LOG", "log": {"level": "FATAL", "message": "Google Analytics Universal Analytics Source Connector will be deprecated due to the deprecation of the Google Analytics Universal Analytics API by Google. This deprecation is scheduled by Google on July 1, 2024 (see Google\u2019s Documentation for more details). Transition to the Google Analytics 4 (GA4) Source Connector by July 1, 2024, to continue accessing your analytics data.\nTraceback (most recent call last):\n  File \"/Users/maxime/devel/code/airbyte/airbyte-integrations/connectors/source-google-analytics-v4/main.py\", line 22, in <module>\n    raise AirbyteTracedException(\nairbyte_cdk.utils.traced_exception.AirbyteTracedException: Google Analytics Universal Analytics Source Connector will be deprecated due to the deprecation of the Google Analytics Universal Analytics API by Google. This deprecation is scheduled by Google on July 1, 2024 (see Google\u2019s Documentation for more details). Transition to the Google Analytics 4 (GA4) Source Connector by July 1, 2024, to continue accessing your analytics data."}}
{"type": "TRACE", "trace": {"type": "ERROR", "emitted_at": 1719341396169.7432, "error": {"message": "Google Analytics Universal Analytics Source Connector will be deprecated due to the deprecation of the Google Analytics Universal Analytics API by Google. This deprecation is scheduled by Google on July 1, 2024 (see Google\u2019s Documentation for more details). Transition to the Google Analytics 4 (GA4) Source Connector by July 1, 2024, to continue accessing your analytics data.", "internal_message": "Google Analytics Universal Analytics Source Connector will be deprecated due to the deprecation of the Google Analytics Universal Analytics API by Google. This deprecation is scheduled by Google on July 1, 2024 (see Google\u2019s Documentation for more details). Transition to the Google Analytics 4 (GA4) Source Connector by July 1, 2024, to continue accessing your analytics data.", "stack_trace": "Traceback (most recent call last):\n  File \"/Users/maxime/devel/code/airbyte/airbyte-integrations/connectors/source-google-analytics-v4/main.py\", line 22, in <module>\n    raise AirbyteTracedException(\nairbyte_cdk.utils.traced_exception.AirbyteTracedException: Google Analytics Universal Analytics Source Connector will be deprecated due to the deprecation of the Google Analytics Universal Analytics API by Google. This deprecation is scheduled by Google on July 1, 2024 (see Google\u2019s Documentation for more details). Transition to the Google Analytics 4 (GA4) Source Connector by July 1, 2024, to continue accessing your analytics data.\n", "failure_type": "config_error", "stream_descriptor": null}}}
```

## User Impact
We intend to merge this late on Sunday. At that point:
* The users won't be able to configure new `Google Analytics (Universal Analytics)` sources
* Cloud users will be updated to the newest versions which will just raise the config error so no data will be synced
* OSS users will still be able to run the syncs but this will most likely raise system_errors

## Can this PR be safely reverted and rolled back?
- [X] YES 💚 (but we expect that the API Connectors oncall person will receive pager alerts)
- [ ] NO ❌

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-analytics-v4-service-account-only/metadata.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 9e28a926-8f3c-4911-982d-a2e1c378b59c
-  dockerImageTag: 0.0.2
+  dockerImageTag: 0.1.0
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-google-analytics-v4-service-account-only/metadata.yaml:16]: @alafanechere does our release process still supports connectors without `pyproject.toml`?

**Comment by maxi297** [comment]
@girarda you will be the one to merge that. Do you have the almighty powers to ignore failed CI items?

**Comment by maxi297** [comment]
@alafanechere is there a way to disable nightly builds so that we don't get notified in #connector-build-statuses

**Comment by alafanechere** [comment]
> @alafanechere is there a way to disable nightly builds so that we don't get notified in #connector-build-statuses

If you modify the support level from `certified` to `community` the connector won't be part of the nightly builds anymore.

File: airbyte-integrations/connectors/source-google-analytics-v4-service-account-only/metadata.yaml (hunk 1 above)
Comment (alafanechere) [reply airbyte-integrations/connectors/source-google-analytics-v4-service-account-only/metadata.yaml:16]: Yes

File: airbyte-integrations/connectors/source-google-analytics-v4-service-account-only/metadata.yaml [hunk 2]
```diff
@@ -22,9 +22,9 @@ data:
   name: Google Analytics (Universal Analytics)
   registries:
     cloud:
-      enabled: true
+      enabled: false
     oss:
-      enabled: true
+      enabled: false
   releaseStage: generally_available
   supportLevel: community
```
Comment (girarda) [inline airbyte-integrations/connectors/source-google-analytics-v4-service-account-only/metadata.yaml:29]: shouldn't we set the supportLevel to archived [like we did with other connectors we archived](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-appstore-singer/metadata.yaml#L29)?

File: airbyte-integrations/connectors/source-google-analytics-v4/metadata.yaml [hunk 3]
```diff
@@ -29,9 +29,9 @@ data:
     cloud:
       enabled: false
     oss:
-      enabled: true
+      enabled: false
   releaseStage: generally_available
-  supportLevel: certified
+  supportLevel: community
```
Comment (girarda) [inline airbyte-integrations/connectors/source-google-analytics-v4/metadata.yaml:34]: shouldn't we set the supportLevel to archived [like we did with other connectors we archived](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-appstore-singer/metadata.yaml#L29)?

**Review by girarda — APPROVED** [review]

**Comment by octavia-approvington** [comment]
Myomoto says it looks good <br/> thats a niiice

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**